src/site.rkt

```diff
@@ -219,7 +221,7 @@
                                      (img ((src ,(gravatar-image-url (session-email session)
                                                                      48))))
                                      " "
-                                     ,(session-email session)
+                                     ,(session-email session) ;; display name ??
```

Comment:
Hmm yeah, could obfuscate here as well... Then people can get used to obfuscated version for themselves as well. If you prefer leaving it alone though, that seems fine too.

Reply:
That makes sense, and also, if we do add a preference, it would be good to communicate which option was chosen.

I haven't figured out how to test the functionality that requires logging in, though. I created a user file in `(build-path (find-system-path 'addon-dir) "pkg-website-test-users"))`, but when I tried to log in, my browser told me:

```
The application raised an exception with the message:

tcp-connect: connection failed
  hostname: localhost
  port number: 8445
  system error: Connection refused; errno=111

Stack trace:

do-ssl-connect at:
  line 1735, column 0, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/openssl/mzssl.rkt
<unknown procedure> at:
  line 555, column 3, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/racket/contract/private/arrow-val-first.rkt
http-conn-open! at:
  line 67, column 0, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/net/http-client.rkt
http-conn-open at:
  line 274, column 0, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/net/http-client.rkt
<unknown procedure> at:
  line 555, column 3, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/racket/contract/private/arrow-val-first.rkt
http://getpost-impure-port at:
  line 202, column 0, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/net/url.rkt
getpost-pure-port at:
  line 272, column 0, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/net/url.rkt
<unknown procedure> at:
  line 489, column 18, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/racket/contract/private/arrow-val-first.rkt
simple-json-rpc! at:
  line 17, column 0, in file /home/philip/code/tmp/racket-pkg-website/src/json-rpc.rkt
create-session-after-authentication-success! at:
  line 376, column 0, in file /home/philip/code/tmp/racket-pkg-website/src/site.rkt
login-or-register-flow* at:
  line 332, column 0, in file /home/philip/code/tmp/racket-pkg-website/src/site.rkt
<unknown procedure> at:
  line 375, column 33, in file /gnu/store/hmschb8wmw1lfa1x8gdim03lxwxmfnsd-racket-vm-cs-8.8/opt/racket-vm/collects/racket/contract/private/arrow-higher-order.rkt
<unknown procedure> at:
  line 63, column 2, in file /gnu/store/c9l5fwhfmdspnyn5ch2g522ssxkh530a-racket-8.8/lib/racket/pkgs/web-server-lib/web-server/dispatchers/dispatch-servlets.rkt
<unknown procedure> at:
  line 141, column 2, in file /gnu/store/c9l5fwhfmdspnyn5ch2g522ssxkh530a-racket-8.8/lib/racket/pkgs/web-server-lib/web-server/private/dispatch-server-with-connect-unit.rkt
```

Reply:
Ah hmm, yeah, it's a bit tricky to set up locally... I think I had to create special local configs to point everything correctly after staring at code and the info at https://github.com/tonyg/racket-pkg-server for a while... 😅 

Here are the local config files I've used in the past, in case that helps: https://gist.github.com/jryans/a5f311e874c6a5ca1104acdf4fbbeb1c